src/App.tsx

```diff
@@ -167,46 +171,22 @@ function App() {
   // ... if we don't wait we'll ALWAYS fire API calls via JsonRpc because provider has not
   // ... been reloaded within App.
   useEffect(() => {
-    if (hasCachedProvider()) {
-      // then user DOES have a wallet
-      connect().then(() => {
-        setWalletChecked(true);
-        trackGAEvent({
-          category: "App",
-          action: "connect",
-        });
-      });
```

Comment:
@appleseed-iii  - do you know what this was previously tracking? Seems like it would have fired a ton. 
Was the intent to track when a user connected their wallet? This seems like it would have fired every time a user loaded the page, regardless of if they manually connected. Depending on what we want to do, we can move this outside of useEffect and either:
- Track the connect button click event 
- check if isConnected = true , and if true, fire the event.

Reply:
yea it would have fired on every refresh, but not literally every page load.

I think tracking the connect button click event is more useful